### PR TITLE
649 Publish donor import file schema as nuget package

### PR DESCRIPTION
**Pull request by zabeen**
This change is [Reviewable](https://reviewable.io/reviews/Anthony-Nolan/Atlas/864)

### CODE REVIEW

**Review by zabeen — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/Anthony-Nolan/Atlas/864)* status: 0 of 47 files reviewed, 1 unresolved discussion

*[a discussion](https://reviewable.io/reviews/Anthony-Nolan/Atlas/864#-NM-5Htp2Gok7JpuFkb_:-NM-5Htq9YC7SQEeYvNA:b-h0wxp1) (no related file):*
This PR includes changes from previous PR https://github.com/Anthony-Nolan/Atlas/pull/863
Wait to review until after 863 is merged.

Also, the first commit on this review contains lots of namespace changes, so may want to review by commit.

___

**Review by daria-sorokina-da — APPROVED** [review]
> Reviewed 45 of 47 files at r1, 4 of 4 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/Anthony-Nolan/Atlas/864)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @zabeen)

**Pull request merged**
